### ISSUE TITLE
kapt is not listed as a dependency in Installation paragraph

### ISSUE THREAD

**Issue by AlexBonel** [arunkumar9t2/scabbard#29]
There's no info about kapt presence in a project on [Installation](https://arunkumar9t2.github.io/scabbard/#installation) paragraph. If you try to use the plugin in a project without `kotlin-kapt` you get an error:

```
A problem occurred evaluating script.
> Configuration with name 'kapt' not found.
```

I think it's better to explicitly list `kapt` as a requirement in this section.

**Comment by arunkumar9t2** [comment]
Thanks for reporting. `kapt` is not actually need. I have some changes planned as part #15 . Will address this for next release. I am assuming you have tried to apply this on Kotlin multimodule project, if that is the case then please consider trying this [solution](https://github.com/arunkumar9t2/scabbard/issues/17#issuecomment-570633460).

**Comment by AlexBonel** [comment]
Sorry for delayed answer. I've tried to apply this on pure Java project and it doesn't work due to the issue 😃

**Comment by arunkumar9t2** [comment]
This issue was fixed as part of #15 and will be available in 0.2.0.
